lua/git-worktree/init.lua

```diff
@@ -217,10 +241,11 @@ end
 
 M.switch_worktree = function(path)
     status:reset(2)
+    status:status("Switching Worktree")
     has_worktree(path, function(found)
 
         if not found then
-            error("worktree does not exists, please create it first")
+            status:error("worktree does not exists, please create it first " .. path)
```

Comment:
`status:error` iirc does not print error, which wont stop the execution here.

Reply:
status:error calls error in my version of status.lua ??